draft-vvv-webtransport-quic.md

```diff
@@ -129,23 +131,70 @@ the client receives a TLS Finished message from the server.
 
 In order to identify itself as a WebTransport application, QuicTransport relies
 on TLS Application-Layer Protocol Negotiation {{!RFC7301}}.  The user agent MUST
-request the ALPN value of "wq" and it MUST NOT establish the session unless that
-value is accepted.
+request the ALPN value of "wq-draft01" and it MUST NOT establish the session
+unless that value is accepted.
+
+## Client Indication
+
+In order to verify that the client's origin is allowed to connect to the server
+in question, the user agent has to communicate the origin to the server.  This
+is accomplished by sending a special handshake message, called *client
```

Comment:
I wouldn't call this a handshake message because it could be interpreted as meaning QUIC/TLS handshake

Reply:
Removed word "handshake".